### PR TITLE
AA-4660 / dex swaps upd

### PR DESCRIPTION
**Pull request by forgxyz**
Parses swaps through aggregator and unions with existing swaps model in `swaps_final`.

Note - the new prices release includes token contract, which will make adding in price data possible. The new `ez_dex_swaps` model with these fields will be released when the pricing data goes live in May. This just fixes the missing swaps.

Swaps models moved to new directory `silver/defi/`

### CODE REVIEW

File: models/silver/defi/silver__swaps_aggregator.sql [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                DISTINCT tx_id
+            FROM
+                parse_swap_log
+        )
+        AND block_height >= 67100587
+
+{% if is_incremental() %}
+AND _inserted_timestamp >= (
+    SELECT
+        MAX(_inserted_timestamp)
```
Comment (WHYTEWYLL) [inline models/silver/defi/silver__swaps_aggregator.sql:77]: modified

File: models/silver/defi/silver__swaps_final.sql [hunk 2]
```diff
@@ -0,0 +1,100 @@
+{{ config(
+    materialized = 'incremental',
+    incremental_strategy = 'merge',
+    merge_exclude_columns = ["inserted_timestamp"],
+    cluster_by = ['block_timestamp::DATE'],
+    unique_key = 'swaps_final_id',
+    tags = ['scheduled', 'streamline_scheduled', 'scheduled_non_core']
```
Comment (WHYTEWYLL) [inline models/silver/defi/silver__swaps_final.sql:7]: scheduled and scheduled_non_core?

File: models/gold/defi/defi__ez_swaps.sql [hunk 3]
```diff
@@ -83,15 +81,9 @@ SELECT
         swaps_id,
         {{ dbt_utils.generate_surrogate_key(['tx_id', 'swap_index']) }}
     ) AS ez_swaps_id,
-    COALESCE (
```
Comment (eric-laurello) [inline models/gold/defi/defi__ez_swaps.sql:86]: would we still have some nulls on the inserted/modified from the cw data?

Comment (forgxyz) [reply models/gold/defi/defi__ez_swaps.sql:86]: Yes, probably. Good catch

**Review by WHYTEWYLL — APPROVED** [review]

**Pull request merged**